data/sql/updates/pending_db_world/rev_1633027575141830731.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1633027575141830731');
+
+DELETE FROM `spell_script_names` WHERE `spell_id` = 15712 AND `ScriptName` = "spell_item_linken_boomerang";
```

Comment:
Use single quotes (' ') instead of quotes (" ") for strings.

```suggestion
DELETE FROM `spell_script_names` WHERE `spell_id` = 15712 AND `ScriptName` = 'spell_item_linken_boomerang';
```